### ISSUE TITLE
Problems together with Alexa Actions Skill, seems like the spoken text is not updated?

### ISSUE THREAD

**Issue by dreimer1986** [keatontaylor/alexa-actions#102]
###  For all support requests it is strongly recommend you join the #alexa-actionable-notifications discord channel for assistance. Myself and others are able to help you in real-time with issues.  [Discord Invite Link](https://discord.gg/AWgE6g)

For all other issues and feature requests, please replace this text with your issue or feature request. Thank you so much for all the support!

- Nice that you recommend the Discord Server, but the invite link is dead ^^
- Knowing that this repo seems to be unmaintained by now, maybe someone has a idea how to fix it anyways.
- I already tried my luck in the repo of Alexa Media Player, but even though the code talking to the stuff here is over there, the maintainer seems to be not interested in it anymore. At least he is unable to help with that code part... https://github.com/custom-components/alexa_media_player/issues/1486

**Describe the bug**
I am tinkering with Alexa Actions for a while already. Out of circumstances I wanna fix with some fun (Parents showing up in my flat when I am not at home for cleaning and complaining) I updated the skill to allow as reply ANY STRING Alexa is hearing. I abused AMAZON.SearchQuery for that. This part works flawless already, but what does not work is stacking Actions. First let me show you what I did:

Automations:
```
- id: telegram-bot-to-alexa
  alias: Telegram Bot Textausgabe auf Alexa
  trigger:
  - platform: event
    event_type: telegram_text
  action:
  - service: script.activate_alexa_actionable_notification
    data:
      text: '{{ trigger.event.data.text }}'
      event_id: alexa_notification_telegram-bot
      alexa_device: media_player.daniels_echo_show
  - service: notify.telegrambot
    data:
      message: '{{ trigger.event.data.text }}'
  mode: single
- id: alexa-to-telegram-bot
  alias: Alexa auf Telegram Bot Textausgabe
  trigger:
  - platform: event
    event_type: alexa_actionable_notification
    event_data:
      event_id: alexa_notification_telegram-bot
  action:
  - service: notify.telegrambot
    data:
      message: '{{ trigger.event.data.event_response }}'
  mode: single
```

I use a Bot driven Telegram channel for notifications, Surveillance Cam pics etc for a while already. Now I added these two automations to be able to fire phrases which Alexa then is saying. Then I catch the reply Alexa is hearing and post it back thx to the bot. 

The second part works super fine, so I decided to post the problem here as report, not over there in the repo. These text messages Alexa has to say are never passing the skill, so it should be a problem here.

Sidenote, this is for debugging and to verify that my written text comes through. It will be disabled later:

```
  - service: notify.telegrambot
    data:
      message: '{{ trigger.event.data.text }}'
```

Now after the long introduction... where is the problem? Simple.

![image](https://user-images.githubusercontent.com/29841368/148651409-f0fb67be-82ae-42b3-9ab1-bb895fda7c73.png)

Lets start in the middle at "Test für Github" sent by me to Alexa. Telegram notification verifies that it got through. Alexa says "Test für Github". Nice. My reply "test zurück" is what I was saying and it got passed through. Fine, too. Then I added another one directly after that called "Test zwei für Github" sent by me to Alexa. Telegram notification verifies that it got through. Alexa says "Test für Github".... errr, wrong. It's funny that after some waiting it uses the new line, but if they are sort of stacked together, things go down the drain.

**To Reproduce**
Difficult to get separated from the text above. So please see above.

**Expected behavior**
The correct and recent text to be spoken by Alexa.

**Screenshots**
See above.

**System details**
 - Home-assistant (version): 2021.12.8
 - Hassio (Yes/No): Yes
 - alexa_media  (version from `const.py` or HA startup): 3.10.15
 - alexapy (version from `pip show alexapy` or HA startup): 1.25.3
 - Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). Yes

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component). We'll be most likely asking for them anyway.
[home-assistant.txt](https://github.com/custom-components/alexa_media_player/files/7833571/home-assistant.txt)

**Additional context**

EDIT: And another sidenote. Yesterday I listened to some music via Spotify on Alexa and before that I tinkered with that very same problem. I have a automation asking me if I wanna switch on the light when the light sensor detects darkness.

```
- id: nachts-lampen-an
  alias: Nachts Lampen an
  trigger:
  - platform: time_pattern
    minutes: '/5'
  condition:
  - condition: and
    conditions:
    - condition: state
      entity_id: device_tracker.hd1917
      state: 'home'
    - condition: time
      after: '16:00:00'
      before: '22:00:00'
    - condition: numeric_state
      entity_id: sensor.hue_motion_illuminance
      below: '8000'
    - condition: not
      conditions:
      - condition: state
        entity_id: media_player.plex_plex_for_windows_reimi_pc
        state: 'playing'
      - condition: state
        entity_id: timer.leaving
        state: 'active'
      - condition: state
        entity_id: timer.night_lamps
        state: 'active'
      - condition: state
        entity_id: light.wohnzimmer
        state: 'on'
  action:
  - service: script.activate_alexa_actionable_notification
    data:
      text: Es wird dunkel. Lampen an?
      event_id: actionable_notification_nachts_lampen
      alexa_device: media_player.daniels_echo_show
  - service: notify.telegrambot
    data:
      message: Es wird dunkel. Lampen an?
  mode: single
- id: nachts-lampen-an-yes
  alias: Lamps on after saying so
  trigger:
  - platform: event
    event_type: alexa_actionable_notification
    event_data:
      event_id: actionable_notification_nachts_lampen
      event_response: ResponseYes
  action:
  - service: homeassistant.turn_on
    entity_id: light.esszimmer
  - service: homeassistant.turn_on
    entity_id: light.wohnzimmer
  mode: single
- id: nachts-lampen-an-no
  alias: Lamps stay off after saying so
  trigger:
  - platform: event
    event_type: alexa_actionable_notification
    event_data:
      event_id: actionable_notification_nachts_lampen
      event_response: ResponseNo
  - platform: event
    event_type: alexa_actionable_notification
    event_data:
      event_id: actionable_notification_nachts_lampen
      event_response: ResponseNone
  action:
  - service: timer.start
    target:
      entity_id: timer.night_lamps
  mode: single
```

I did not even realize it was asking that question, because due to the former tinkering it never said "Es wird dunkel. Lampen an?" but one of the lines of my testing, here "Test"

**Comment by dreimer1986** [comment]
Well, it's a duplicate of #75
